BrainPortal/lib/cbrain_checker.rb

```diff
@@ -76,5 +76,21 @@ def self.check(checks_to_run, options={})
 
   end
 
+  protected
+
+  # helper method to runs a query and checks that type column contains
+  # already loaded objects; on mild issues (forgot run rake install etc...)
+  # in developer env gives warning but fails execution in production or test environment
+  def self.validate_sti(query)
+    table = query&.table_name || 'a table'
```

Comment:
table should never be blank, and .table_name should never be invoked on a nil so that line should be made straighforwardly:

```ruby
  table = query.table_name
```